addons/mail/static/src/models/dialog/dialog.js

```diff
@@ -15,6 +18,40 @@ registerModel({
         hasElementInContent(element) {
             return Boolean(this.record && this.record.containsElement(element));
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeAttachmentViewer() {
+            if (this.attachmentListOwnerAsAttachmentView) {
```

Comment:
name inconsistency Attachment**Viewer**/AsAttachment**View**

Reply:
Don't have a proper naming for viewer / as view alternative... Note that the "asX" introduced here was necessary for making dialog models with delete attachment, so that the owner were the same but flow differs.

The owner won't be the same now that we changed it here to attachmentList, as the delete will be on attachment card/image. We could just drop the "asX" for the time being. One benefit of dropping it is shorter and more readable fields.